internal/tool/multi/deployer.go

```diff
@@ -133,7 +133,9 @@ func newDeployer(ctx context.Context, deploymentId string, config *MultiConfig)
 	}
 
 	// Create the trace saver.
-	traceDB, err := perfetto.Open(ctx, perfettoFile)
+	traceDB, err := perfetto.Open(ctx, perfettoFile, perfetto.DBOptions{
+		MaxTraceBytes: 100 * 1024 * 1024, // 100MB
```

Comment:
Is there some reasoning we can write down in a comment for picking this size? Perhaps based on every trace size?

Reply:
Done.